### PR TITLE
fix: ui issues and improvements pt. 2

### PR DESCRIPTION
**Pull request by mgaseta**
# Description
Fixes partially #1388 
Fix issues: 
- [Action button on details page](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225604402)
- [White stripe on the bottom of the pages](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225615607)
- [Keep side menu active](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225619966)
- [Adjusting client modal fields](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225648824)
- [Client search modal selected style](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225655190)
- [Client search modal other fixes](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225661431)
- [Seedlot Registration Step 5 margins](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225720122)
- [Loader covering side menu](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225743906)
- [Back to step one link on seedlot submit modal](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225756416)
- [Delete SPZ button actions style](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225773694)
- [Date fields on extract and storage step on the review form](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225780755)
- [Pagination values on the seedlot review table](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225789250)
- [Go to seedlot error handling](https://github.com/bcgov/nr-spar/issues/1388#issuecomment-2225798075)

Other issues fixed:
- Breadcrumbs cursor to pointers
- Seedlot tables skeletons hiding headers

#### New
- Lots of styles :)

#### Changed
- Made total-counts header visible on the BE
- Styles on the client search modal
- Validation for current menu on the header
- Notification component for submit modal
- Step 6 date fields on review 
- Action button on seedlot details page
- Minor texts and styles

#### Removed
- Nothing :)

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-8-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1458-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1458-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-8-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1458-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1458-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

**Review by ngunner15 — APPROVED** [review]
LGTM💯

File: frontend/src/components/SeedlotRegistrationSteps/ExtractionAndStorageStep/constants.ts [hunk 1]
```diff
@@ -50,8 +50,8 @@ export const inputText = {
     },
     storage: {
       labelText: {
-        start: 'Storage start date (optional)',
-        end: 'Storage end date (optional)'
+        start: `Storage start date${!isReview ? ' (optional' : ''}`,
+        end: `Storage end date${!isReview ? ' (optional' : ''}`
```
Comment (craigyu) [inline frontend/src/components/SeedlotRegistrationSteps/ExtractionAndStorageStep/constants.ts:54]: missing closing brackets :) for `(optional`

**Review by craigyu — APPROVED** [review]
NICE, should we get @mmarsoleta to review and approve the UI changes?

**Comment by mmarsoleta** [comment]
> NICE, just one typo
> 
> should we get @mmarsoleta to review and approve the UI changes?

sure, just let me know when I can review it again, and the link please 🥳

File: frontend/src/components/SeedlotRegistrationSteps/ExtractionAndStorageStep/constants.ts (hunk 1 above)
Comment (mgaseta) [reply frontend/src/components/SeedlotRegistrationSteps/ExtractionAndStorageStep/constants.ts]: Hawk eyed!

Fixed here: https://github.com/bcgov/nr-spar/pull/1458/commits/df86e4733228ed31d8b5f0400105869d7acdedbb

**Comment by mgaseta** [comment]
Talked with @mmarsoleta and everything is fine! Merging it!

**Pull request merged**